modules/aspect_rules_rollup/0.15.0/MODULE.bazel

```diff
@@ -0,0 +1,20 @@
+"aspect-build/rules_rollup"
+
+module(
+    name = "aspect_rules_rollup",
+    compatibility_level = 1,
+    version = "0.15.0",
+)
+
+# Lower-bound (minimum) versions of direct dependencies
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+# Need #397 to skip stardoc targets
+bazel_dep(name = "aspect_bazel_lib", version = "1.30.2")
+# Need attribute 'dev' in 'npm_package_store_internal' rule
+bazel_dep(name = "aspect_rules_js", version = "1.19.0")
+bazel_dep(name = "rules_nodejs", version = "5.5.3")
+bazel_dep(name = "platforms", version = "0.0.5")
+
+rollup = use_extension("@aspect_rules_rollup//rollup:extensions.bzl", "rollup")
+rollup.toolchain(name = "rollup", rollup_version = "v2.70.2")
```

Comment:
@gregmagolan note, I think this will fall into our existing problem of making it hard for users to pick a different version